source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+the :file:`$HOME/telemetrics-backend/scripts` directory:
+
+.. code-block:: console
+
+   ./deploy.sh -H localhost -a install -d clr
+
+The script will start and list all the defined options and prompt you for the
+:guilabel:`PostgreSQL` database password as shown below:
+
+.. code-block:: console
+
+   Options:
+     host: localhost
+     distro: clr
+     action: install
+     repo: https://github.com/clearlinux/telemetrics-backend
+     source: master
+     type: git
+   DB password: (default: postgres):
+
+For the :guilabel:`DB password:`, press the :kbd:`Enter` key to accept the
+default password `postgres`. The :command:`swupd` begins installing the
+required software bundles to set up the telemetrics backend server. The output
+will look similar to what is shown below:
+
+.. code-block:: console
+
+   swupd-client bundle adder 3.12.7
+      Copyright (C) 2012-2017 Intel Corporation
+
+   Downloading packs...
+
+   Extracting application-server pack for version 18740
+        ...5%
+   Extracting database-basic-dev pack for version 18670
+        ...10%
+   Extracting database-basic pack for version 18670
+        ...15%
+   Extracting os-clr-on-clr pack for version 18740
+        ...21%
+   Extracting sysadmin-basic-dev pack for version 18740
+        ...26%
+   Extracting storage-utils-dev pack for version 18770
+        ...31%
+   Extracting os-core-update-dev pack for version 18760
+        ...36%
+   Extracting network-basic-dev pack for version 18760
+        ...42%
+   Extracting mixer pack for version 18790
+        ...47%
+   Extracting os-installer pack for version 18800
+        ...52%
+   Extracting mail-utils-dev pack for version 18760
+        ...57%
+   Extracting koji pack for version 18800
+        ...63%
+   Extracting go-basic pack for version 18800
+        ...68%
+   Extracting dev-utils-dev pack for version 18820
+        ...73%
+   Extracting python-basic-dev pack for version 18750
+        ...78%
+   Extracting perl-basic-dev pack for version 18610
+        ...84%
+   Extracting c-basic pack for version 18800
+        ...89%
+   Extracting os-core-dev pack for version 18800
+        ...94%
+   Extracting web-server-basic pack for version 18680
+        ...100%
+   Installing bundle(s) files...
+        ...100%
+   Calling post-update helper scripts.
+   Possible filedescriptor leak : 8 (socket:[30833])
+   Bundle(s) installation done.
+
+.. note::
+
+   This script uses :command:`sudo` to run commands and you may be prompted to
+   enter your user password at any time while the script is executing. If this
+   occurs, enter your user password to execute the :command:`sudo` command.
+
+   .. code-block:: console
+
+      Password:
+
+   You may also see an informational message about setting the
+   :guilabel:`https_proxy` environment variable if this variable isn't set.
+
+Once the :command:`swupd` command is complete, the script begins processing
```

Comment:
Run-on sentence and subject-verb congruence issue:
Once the :command:`swupd` command is complete, the script begins processing the requirements to install and implement the telemetrics server. Finally, the script enables the server and provides output similar to:

Reply:
done